packages/react-native/src/primitives/IconButton/styles.ts

```diff
@@ -1,7 +1,12 @@
 import { StyleSheet } from 'react-native';
+
+import { StrictTheme } from '../../theme';
 import { IconButtonStyles } from './types';
 
-export const styles: IconButtonStyles = StyleSheet.create({
-  pressed: { opacity: 0.6 },
-  text: { alignSelf: 'center' },
-});
+export const getThemedStyles = (theme: StrictTheme): IconButtonStyles => {
+  const { components, opacities } = theme.tokens;
+
+  return StyleSheet.create({
+    pressed: { opacity: opacities[60], ...components?.iconbutton.pressed },
```

Comment:
Thinking we should probably also provide a `container` prop so users can apply the theme to the `Pressable` when its not pressed

Reply:
+1 Also disabled and size.